### DISCUSSION TITLE
SSD1963 JPG IMAGE PROBLEM

### DISCUSSION THREAD

**Discussion by nodoubtman** [Bodmer/TFT_eSPI#2923] [Q&A - General]
Hello. I have got a new display to play with it is the SSD1963.
I tried the example JPEGDecoder library LittleFS From the web.
And the sd card example from TJpg_Decoder.

Same effect, the jpg is in very low resolution, comparing if i take the ili9488 smaller screen.

Is it normal?
What can i do to normalize the resolution of the jpg ?

Thank You.
Marc.

**Comment by Bodmer** [comment]
The libraries do not limit the jpg image x,y pixel size. You can create jpeg files that have the resolution you want using programs like GIMP or MS Paint etc.

The examples just include jpegs for common screen sizes.

**Reply by nodoubtman** [reply]
Hello thanks for the response. In windows 10 i see the file it is very beautiful and on the tft screen ssd1963 it is distorted, kinda color fading colors.

Thanks.
Marc.

**Comment by Rollmops67** [comment]
Did you put `tft.setSwapBytes(true);` line in your setup ?? (or if yes, try to change to false)

This may help.

Roland

**Reply by nodoubtman** [reply]
yes, i tried, same result.


thanks.
Marc.

**Reply by nodoubtman** [reply]
:(

**Comment by SP6U** [comment]
I'm currently learning how to program using PlatformIO in VSCode, and I’m working with an ESP32-WROOM-32D (38-pin) development board.

I’d like to connect a 7-inch TFT display with the SSD1963 controller and a touchscreen, but I'm not sure how to set it up correctly — both in terms of hardware and software.

Does anyone have a working and tested example project for this combination (ESP32 + SSD1963 with touch support), preferably including:

Wiring diagram or connection details

Initialization code for the display and touch controller

Recommended libraries and drivers

Touchscreen setup (resistive or capacitive)

What I’m really looking for is a complete PlatformIO project folder — with all required libraries and drivers already included — that I can simply unzip, open in VSCode, and upload to my ESP32 without any extra setup.

If someone could share such a ready-to-run ZIP file, I’d be extremely thankful. That would help me a lot in learning and getting started. I’ll take care of everything else from there.

Thanks so much in advance for your support!
